File: SourceCode/CSharp/Common/Logger.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Common
{
    public class Logger
    {
        string filePath = @"C:\Users\Rameshbabu-P\Documents\GitHub\Log\PE.log";
        public void LogMessage(string message)
        {
            try
            {
                StreamWriter sw = new StreamWriter(filePath, true);
                string shortDate = DateTime.Now.ToShortDateString();
                string shortTime = DateTime.Now.ToShortTimeString();
                sw.WriteLine(string.Format("{0} {1} {2}", shortDate, shortTime, message));
                sw.Flush();
                sw.Close();
            }
            catch (Exception ex)
            {
                throw new Exception(ex.Message);
            }
        }
    }
}

File: SourceCode/Java/Common/src/Logger.java
import java.io.*;
import java.util.Date;

public class Logger 
{
	public static void LogMessage(String logMessage)
	{
		File myFile = new File("C:/Users/Rameshbabu-P/Documents/GitHub/Log/","PATEXN.log");
		try 
		{
			if(!myFile.exists())
			{
				myFile.createNewFile();
			}
			
			FileWriter fileWriter=new FileWriter(myFile,true);
			BufferedWriter bf = new BufferedWriter(fileWriter);
			Date date = new Date();
			String datetime=date.toString();
			bf.newLine();
			bf.write(datetime + "  " + logMessage);
			bf.newLine();
			bf.flush();
			bf.close();
		} 
		catch (IOException e) 
		{
			e.printStackTrace();
		}
	}
}

